generators/app/templates/src/app/screens/RecoverPassword/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useCallback } from 'react';
+
+import RecoverPassword from './layout';
+
+function RecoverPasswordContainer() {
+  const handleSubmit = useCallback((event: React.FormEvent<Element>) => {
+    event.preventDefault();
+    // TODO: Implement
+  }, []);
+
+  const handleEmailChange = useCallback((event: React.FormEvent<Element>) => {
+    event.preventDefault();
```

Comment:
added this to avoid ts from alerting errors on unused params !

Reply:
Can't this be configured to be ignored in some config file (may be `tsconfig.json`) instead of adding a line of code?